src/universal/components/OutcomeCard/OutcomeCardFooter.js

```diff
@@ -114,7 +120,7 @@ const OutcomeCardFooter = (props) => {
       </div>
       <div className={styles.buttonBlock}>
         <button className={buttonStyles} onClick={handleStatusClick}>
```

Comment:
is that something we wanna support? 
changing teams means no history, since other members may not be on the other team.
that might get really complicated with the summary